data/tooling-data.yaml

```diff
@@ -2409,3 +2409,32 @@
   supportedDialects:
     draft: ['4', '6', '7']
   lastUpdated: '2024-09-03'
+
+- name: justinrainbow/json-schema
+  description: 'A PHP Implementation for validating JSON Structures against a given Schema with support for Schemas of Draft-3 or Draft-4.'
+  toolingTypes: ['validator']
+  languages: ['PHP']
+  environments: ['PHP >= 7.2']
```

Comment:
Can you please remove the environment property and provide it in the toolingListingNotes? This is reserved for values like:
                "Web (Online)",
                "GitHub Actions",
                "Command Line",
                "COM/ActiveX"

Reply:
I have made the required changes.